mungegithub/mungers/matchers/comment/interactions.go

```diff
@@ -41,23 +41,27 @@ type CommandName string
 
 // Match will return true if the comment is a command with the given name
 func (c CommandName) Match(comment *github.IssueComment) bool {
```

Comment:
Update the comment.  It will return true if the comment's body contains a command at the beginning of a line (other text is permitted on other lines)